certbot/cli.py

```diff
@@ -357,7 +357,8 @@ def _get_help_string(self, action):
     }),
     ("delete", {
         "short": "Clean up all files related to a certificate",
-        "opts": "Options for deleting a certificate"
+        "opts": "Options for deleting a certificate",
+        "usage": "\n\n  certbot delete --cert-name CERTNAME"
```

Comment:
Nice catch. Testing with the versions of argparse I have locally, it doesn't seem to matter, but this is an undocumented implementation detail of `argparse` so I agree that we should be consistent here.